visualization/app/codeCharta/util/simplePipes/SimplePipesModule.ts

```diff
@@ -1,9 +1,10 @@
 import { NgModule } from "@angular/core"
 import { RemoveExtensionPipe } from "./removeExtension.pipe"
-import { MetricDescriptorTitlePipe } from "./metricDescriptorTitle.pipe"
+import { AttributeDescriptorTooltipPipe } from "./attributeDescriptorTooltip.pipe"
+import { TruncateTextPipe } from "./truncateText.pipe"
 
 @NgModule({
-	declarations: [RemoveExtensionPipe, MetricDescriptorTitlePipe],
-	exports: [RemoveExtensionPipe, MetricDescriptorTitlePipe]
+	declarations: [RemoveExtensionPipe, AttributeDescriptorTooltipPipe, TruncateTextPipe],
+	exports: [RemoveExtensionPipe, AttributeDescriptorTooltipPipe, TruncateTextPipe]
 })
 export class SimplePipesModule {}
```

Comment:
[Optional]
I am not too happy with this generic module. It's similar to a generic `utils.helper` with different kinds of helper in it. I prefer an own module for each pipe. But you haven't introduced it, so feel free to refactor it or not.